src/js/pages/Escrow/components/CardEscrowBuyer.jsx

```diff
@@ -107,13 +107,33 @@ PreFund.propTypes = {
   statusContactCode: PropTypes.string
 };
 
+const Dispute = () => (
+  <React.Fragment>
```

Comment:
Just a tip: you can do `<Fragment>` now. (just need to import it above like so: `import React, {Component, Fragment} from 'react';`